File: 2292/code.c
#include <stdio.h>
#include <string.h>
#include <math.h>




int main() {

   int i = 2;
   int j = 7;
   int k =1;
   int count = 2;

   int num;
   scanf("%d", &num);

   while(1){

    if(i<=num && j>=num){

        printf("%d", count);
        break;

    }

    i = i +6*k;
    j = j + 6*(k+1);
    k++;
    count++;
   }
}

File: 2292/code.py
num = int(input())
a = 2
b = 7
k = 1
count = 2
while True :

    if a<=num<=b :
        print(count)
        break

    a = a + 6*k
    b = b + 6*(k+1)
    k = k+1
    count = count+1

